lib/migration-steps/validation.js

```diff
@@ -81,6 +80,64 @@ const getFieldErrors = getStepErrors.bind(null, {
   validations: fieldPropsValidations
 });
 
+const fieldMovementValidations = {
+  toTheTop: Joi.any(),
+  toTheBottom: Joi.any(),
+  afterField: Joi.string().required(),
+  beforeField: Joi.string().required()
+};
+
+const getFieldMovementErrors = function (step) {
+  const validMoves = ['toTheTop', 'toTheBottom', 'afterField', 'beforeField'];
```

Comment:
Shall we derive this from the `Object.keys` of `fieldMovementValidations`?

Reply:
Sounds good

Reply:
changed